src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformAlleleFractionSegmentation.java

```diff
@@ -18,7 +19,7 @@
  * Groups contiguous targets with the same minor allele fraction for a single sample.
  *
  *  <p>
- *     The --tumorHets file is from {@link GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
+ *     The --tumorHets file is from {@link org.broadinstitute.hellbender.tools.exome.GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
```

Comment:
You can import `org.broadinstitute.hellbender.tools.exome` to clean up both javadoc links here.  I think you can set IntelliJ to do this automatically; e.g., https://stackoverflow.com/questions/24266917/how-to-make-intellij-autocompletion-not-insert-the-full-path-in-javadoc.